src/coreComponents/math/CMakeLists.txt

```diff
@@ -2,8 +2,9 @@
 # Specify all headers
 #
 set( math_headers
-  	 interpolation/Interpolation.hpp
-	 extrapolation/Extrapolation.hpp)
+     Constants.hpp
```

Comment:
Maybe sure that this is consistent with the file name that you will choose for the new file